src/arch/xtensa/lib/cpu.c

```diff
@@ -37,6 +37,31 @@ extern struct xtos_core_data *core_data_ptr[PLATFORM_CORE_COUNT];
 
 static uint32_t active_cores_mask;
 
+#ifndef CONFIG_HAVE_SLAVE_CORE_ROM
+extern void *shared_vecbase_ptr;
+extern void _WindowOverflow4;
+
+static void alloc_shared_slave_cores_objects(void)
```

Comment:
Please comment what these functions are doing. Commit message is also non descriptive.

Reply:
both done